prov/efa/src/rxr/rxr_pkt_type.h

```diff
@@ -90,7 +90,19 @@
 #define RXR_READ_TAGRTM_PKT		129
 #define RXR_READ_RTW_PKT		130
 #define RXR_READ_RTR_PKT		131
-#define RXR_EXTRA_REQ_PKT_END		132
+
+#define RXR_DC_REQ_PKT_BEGIN		132
+#define RXR_DC_EAGER_MSGRTM_PKT 	133
+#define RXR_DC_EAGER_TAGRTM_PKT 	134
+#define RXR_DC_MEDIUM_MSGRTM_PKT 	135
+#define RXR_DC_MEDIUM_TAGRTM_PKT 	136
+#define RXR_DC_LONG_MSGRTM_PKT  	137
+#define RXR_DC_LONG_TAGRTM_PKT  	138
+#define RXR_DC_EAGER_RTW_PKT    	139
+#define RXR_DC_LONG_RTW_PKT     	140
+#define RXR_DC_WRITE_RTA_PKT    	141
+#define RXR_DC_REQ_PKT_END		142
```

Comment:
Since `RXR_DC_REQ_PKT_END ` is not a specific control type of a protocol, there is no need to make `RXR_EXTRA_REQ_PKT_END ` strictly greater than `RXR_DC_REQ_PKT_END `. `RXR_EXTRA_REQ_PKT_END ` can be updated when extra features are introduced.